tests/legacy-cli/e2e/assets/protractor-saucelabs.conf.js

```diff
@@ -19,33 +19,17 @@ exports.config = {
   multiCapabilities: [
     {
       browserName: 'chrome',
-      version: '41',
+      version: 'latest',
```

Comment:
Can we pin this and have a regular reminder to update it?

We should avoid unpinned depends so that ci doesn't go red unexpectedly

Reply:
Good point. Updated.